pkg/rtc/participant.go

```diff
@@ -1406,7 +1406,7 @@ func (p *ParticipantImpl) onSubscriberOffer(offer webrtc.SessionDescription) err
 }
 
 func (p *ParticipantImpl) removePublishedTrack(track types.MediaTrack) {
-	p.RemovePublishedTrack(track, false)
+	p.RemovePublishedTrack(track, false, true)
```

Comment:
This used to be soft close before, but I think it is fine if this is hard closed.